zkevm-circuits/src/evm_circuit.rs

```diff
@@ -183,26 +183,29 @@ impl<F: Field> EvmCircuit<F> {
             fixed_table_tags: FixedTableTag::iter().collect(),
         }
     }
-
-    pub fn get_test_cicuit_from_block(block: Block<F>) -> Self {
+    #[cfg(feature = "test-circuits")]
+    /// Construct the EvmCircuit with only subset of Fixed table tags required by tests to save
+    /// testing time
+    pub(crate) fn get_test_cicuit_from_block(block: Block<F>) -> Self {
```

Comment:
Here a small typo `cicuit`

Reply:
Nice catch. Fixed in [4277a5b](https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1524/commits/4277a5bcf40a5fdd660480f76df3f4302ec8ad09)